integration/environment/namespace.go

```diff
@@ -38,6 +39,12 @@ func (e *Environment) SetupNamespace() error {
 	return nil
 }
 
+// NamespaceDeletionInProgress returns true if the error indicates deletion will happen
```

Comment:
very cool - so we don't get all those red herrings on stdout?

Reply:
yes, had to make one small fix though :)